src/collections/projects/distributed-performance-management-of-service-meshes/index.mdx

```diff
@@ -2,6 +2,9 @@
 title:  "Distributed Performance Management of Service Meshes"
 thumbnail: ./distributed-performance_green.svg
 published: true
+redirect_from: 
+ - /distributed-performance
+ - /projects/distributed-performance
 ---
```

Comment:
What will the permalink be? 

/projects/distributed-performance-management?

The of-service-meshes portion my too long of a URL.

Reply:
We did discuss about including a permalink in the frontmatter. Views @Tanuj22 @Nikhil-Ladha on having a pernalink field.

Reply:
Should like a good inclusion I think